src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
```

Comment:
u need create three balls, not three Lottery object, u can create 1 object Lottery, and using method getRandomBall(), create 3 balls